frontend/src/requests/SneakersRequest.js

```diff
@@ -9,3 +9,8 @@ export const getAllProductsByPagination = async (page) => {
 	const response = await axios.get(`/sneakers?page=${page}`);
 	return response.data;
 };
+
+export const getRelatedSneakersById = async (tags) => {
+	const response = await axios.get(`${SERVER_DOMAIN}/sneaker/${tags}`);
```

Comment:
Please, update this endpoint, the SERVER_DOMAIN variable has been removed.

Reply:
Done